app/src/main/java/com/adyen/testcards/ui/CopyMenu.kt

```diff
@@ -0,0 +1,56 @@
+package com.adyen.testcards.ui
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.ClipboardManager
+import androidx.compose.ui.platform.LocalClipboardManager
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.unit.dp
+import com.adyen.testcards.R
+
+@Composable
+fun CopyMenu(
```

Comment:
What do you think of using the copy icon next to each item, instead of creating a header?

![image](https://github.com/user-attachments/assets/267b9812-d2c5-4bd1-923d-549b62529dd1)
Got the images from [here](https://m3.material.io/components/menus/overview).

Reply:
Had that before, but the icon is very repetitive and so is prefixing every item with "Copy ${title of your item}". I figured doing it once makes it look less noisy